src/ListGuesser.js

```diff
@@ -63,7 +63,6 @@ const ListGuesser = props => (
 ListGuesser.propTypes = {
   children: PropTypes.oneOfType([PropTypes.node, PropTypes.func]),
   resource: PropTypes.string.isRequired,
-  filters: PropTypes.object.isRequired,
```

Comment:
`filters: PropTypes.object` then right?
Also, maybe passing `false` instead of  `null` would be more explicit?

Reply:
Ok for `null`, and good catch for the `PropTypes.element`! Could you change it then? Thanks!